entregas/patinoSara/carreraCamellos.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+
+public class carreraCamellos {
+    public static void main(String[] args) {
+        Scanner inputUsuario = new Scanner(System.in);
+
+        int posicionJugador = 0;
+        int posicionOrdenador = 0;
+        int numeroTotalCasillas = 60;
+        double acertarAgujero = Math.random();
+        double agujeroOrdenador = Math.random();
+        //String bordes = "---+-----------+---------+---------+---------+---------+-----------+";
+        //String casilla1 = "---| 0.........1.........2.........3.........4.........5.........6 |";
+        //String casilla2 = "---| 0123456789012345678901234567890123456789012345678901234567890 |";
+        //String posicionCamello = "[J]| ;--;'";
+
+        for (int turno = 0; turno <= 50; turno = turno + 1) {
+
+            System.out.println("Por cual agujero lo quieres intentar meter?");
+            int agujero = inputUsuario.nextInt();
+
+            if(posicionJugador>numeroTotalCasillas || posicionOrdenador> numeroTotalCasillas){
+                System.out.println("Se termino el juego");
+            }
```

Comment:
Si asumimos que la gestión la hacemos con for, entonces este if debería servir además para bloquear el resto de la ejecución en caso que se termine el juego.